sygnal.yaml.sample

```diff
@@ -197,6 +197,11 @@ apps:
   #  #
   #  # The default is 'production'. Uncomment to use the sandbox instance.
   #  #platform: sandbox
+  #  #
+  #  # Specifies whether to use a hexadecimally encoded push token. If True
+  #  # (default) the pushkey received is trancoded from base64 to hex. If False
+  #  # the pushkey received is passed to APNs as is.
+  #  #apns_device_token_use_hex: false
```

Comment:
this option seems like it is named the opposite of what it means: I would expect `use_hex = True` to mean that Sygnal expects hex tokens from clients rather than base64.

Is it just me?

Reply:
Hi @reivilibre how about naming it `apns_device_token_transcode_to_hex`?

Reply:
I think the `apns` is probably redundant since it is in the apns pushkin config, maybe `convert_device_token_to_hex`?

I suspect the comment needs to be updated a bit too:

> Whether to convert the device push token from base 64 to hex. Defaults to True, set this to False if your client library provides a push token in hex format.

Reply:
I don't have a strong opinion on the prefix, I'll note that none of the other APNS-specific options have it though.